lc0_main.go

```diff
@@ -33,7 +33,7 @@ var (
 	totalGames int
 	pendingNextGame *client.NextGameResponse
 
-	hostname = flag.String("hostname", "http://testserver.lczero.org", "Address of the server")
+	hostname = flag.String("hostname", "http://162.217.248.187", "Address of the server")
```

Comment:
There is `http://api.lczero.org/` address for that!
Let's use it

Reply:
Done!